index.html

```diff
@@ -262,16 +262,20 @@ <h4>Multikey</h4>
           </p>
 
           <p>
-The `publicKeyMultibase` property represents a Multibase-encoded Multikey
-expression of an Ed25519 public key. The value starts with the two-byte prefix
-`0xed01`, followed by the 32-byte Ed25519 public key data. The combined 34-byte
-value is then encoded using base58-btc (`z`) as the prefix.
+The `publicKeyMultibase` value of the verification method MUST be 35 bytes in
+length and starts with the base-58-btc prefix (`z`), as defined in the
+<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase section</a> of
+[[VC-DATA-INTEGRITY]]. A Multibase-encoded Multikey value follows, which MUST
+consist of a binary value that starts with the two-byte prefix `0xed01`, which
+is the Multikey header for an Ed25519 public key, followed by the 32-byte public
+key data, all of which is then encoded using base-58-btc. Any other encoding
+MUST NOT be allowed.
           </p>
 
           <p class="advisement">
 Developers are advised to not accidentally publish a representation of a private
 key. Implementations of this specification will raise errors if they encounter a
-prefix value other than `0xed01` in a `publicKeyMultibase` value.
+Multikey prefix value other than `0xed01` in a `publicKeyMultibase` value.
```

Comment:
If used, the multikey prefixed need to be defined so as to enable interoperable implementations.

Reply:
The Multikey prefixes for EdDSA public and private keys are defined here: https://pr-preview.s3.amazonaws.com/w3c/vc-di-eddsa/pull/63.html#multikey